src/Avalon/StartGameView.py

```diff
@@ -0,0 +1,37 @@
+import discord
+
+from src.Avalon.Game import Game
+from src.Avalon.Roles.Role import Role
+
+
+class StartGameView(discord.ui.View):
+
+    def __init__(self, roles:list):
+        self.roles = [discord.SelectOption(label = role, value = role) for role in roles]
+        super().__init__()
+
+    @discord.ui.select(cls = discord.ui.UserSelect, max_values= 12)
+    async def select_players(self, interaction, select: discord.ui.UserSelect):
+        return await interaction.response.send_message(f"Selected {', '.join([user.display_name for user in select.values])}!")
+
+    @discord.ui.select(cls = discord.ui.Select,
+                       options = [discord.SelectOption(label = role, value = role) for role in Game.ROLES],
+                       max_values= len(Game.ROLES))
```

Comment:
Why is it 12 earlier?

Reply:
I'm assuming we can't play avalon with more than 12 players. We can adjust as required.